src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -100,6 +100,7 @@
     "_from_pandas_udf_function",
     "input_names",  # for pandas_udtf
     "max_batch_size",  # for pandas_udtf
+    "_registered_object_name",  # db object name if already registered
```

Comment:
nit: remove `if already registered`, replace with something like `object name within Snowflake (post-registration)` or so.

Reply:
If it's None I assume the object has not been registered, if not none then it should be a valid object name?